src/index.js

```diff
@@ -1,9 +1,17 @@
-const { SauceJsonReporter } = require('testcafe-reporter-sauce-json/reporter');
-const { Reporter } = require('./reporter');
 const path = require('path');
+const fs = require('fs');
+const { SauceJsonReporter } = require('./json-reporter');
+const { JobReporter } = require('./job-reporter');
 
 module.exports = function () {
     return {
+        noColors: true,
+        sauceJsonReporter: SauceJsonReporter.newReporter(),
+
+        sauceReportJsonPath: process.env.SAUCELABS_REPORT_JSON_PATH || './sauce-test-report.json',
+        disableSauceUpload: process.env.SAUCELABS_DISABLE_SAUCE_UPLOAD !== undefined,
```

Comment:
Do we need such long verbiage? Or would `SAUCE_` as a prefix suffice?

Reply:
`SAUCE_` was my first idea. Finally I used `SAUCELABS_` to be consistent with the reported name.
It can be changed

Reply:
I think your first haunch to use `SAUCE_` was right, since it's more consistent with what we have everywhere else. 

A few examples:
SAUCE_USERNAME, SAUCE_ACCESS_KEY, SAUCE_VM etc.

So I'd vote for using `SAUCE_` rather than `SAUCELABS_`.